cron.go

```diff
@@ -180,6 +180,9 @@ func (c *Cron) run() {
 
 // Stop the cron scheduler.
```

Comment:
If it's not documented, people are likely to manage their own "is cron running state" needlessly, like:

```Go
if isCronRunning {
    cron.Stop()
    isCronRunning = false
}
```